modules/default-secrets.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * nodes/pods/nodes-pods-secrets.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="default-secrets_{context}"]
+= Default secrets
+
+By default, {product-title} creates the following secrets:
+
+* A `builder` dockercfg secret
+* A `builder` service account token secret
+* A `default` dockercfg secret
+* A `default` service account token secret
+* A `deployer` dockercfg secret
+* A `deployer` service account token secret
+
+// TODO: Do we need to describe what each of these are used for? Are the ALL used for accessing the internal registry? Or something else?
+
+These secrets are used to access the integrated {product-registry}.
+
+// TODO: Do we need a note - is it possible for others to be created by default, if they have any other add-ons or other things enabled?
```

Comment:
@sanchezl Could there be more secrets that are included in this list depending on other add-ons? If so, should we mention that, or leave it be?

Reply:
There are hundreds.